index.html

```diff
@@ -284,37 +351,34 @@ <h3 class="language">CSS</h3>
       </ul>
 
     </div>
+    </div>
+</section>
 
-    <div class="contanct-containter">
+    <div class="contanct-containter ">
       <footer class="contact-info">
         <div class="backgound-image"></div>
-
+        <div class="backgound-image desktop">
           <div class="contact-title">
-            <h2 class="title2">Contact me</h2>
-          </div>
-          <div class="contacts-info">
-            <p>
+            <h2 class="title2 desktop">Contact me</h2>
+            <p class="contacts-info">
```

Comment:
- Kindly, make your contact form like the [Figma design](figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=xmWXLXW7NiSVxIum-0), you added unnecessary margin and the form is not centered 👌👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206552104-347ee778-5b3e-490d-af22-b8e9d022779a.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206552125-1460a970-13a5-4887-9c87-4895073a3e4a.png" width="200">|


😎️